manifests/init.pp

```diff
@@ -194,8 +194,8 @@
 ) {
   $odoo_supported_versions = {
     'Debian' => {
-      '11' => ['14.0', '15.0', '16.0', 'system'],
-      '12' => ['14.0', '15.0', '16.0', '17.0'],
+      '11' => ['14.0', '15.0', '16.0', '17.0', '18.0', 'system'],
```

Comment:
CI shows that on Debian 11, python is too old to install odoo 17, and apt is too old to install odoo 18.

For now, odoo 18 seems only usable on Debian 12, so we can revert this line:

```suggestion
      '11' => ['14.0', '15.0', '16.0', 'system'],
```

Reply:
Oh crapbaskets I did it again despite you already telling me.
Ballmer peak is a lot of baloney